common/formatting/unwrapped_line.h

```diff
@@ -74,6 +74,9 @@ class UnwrappedLine {
   // Extends PreFormatToken range spanned by this UnwrappedLine by one token.
   void SpanNextToken() { tokens_.extend_back(); }
 
+  // Extends PreFormatToken range spanned by this UnwrappedLine by one token.
```

Comment:
Distinguish this comment from the one from `SpanNextToken`.
e.g. "... by one token on at the front" vs. "at the back".

Reply:
Done